kernels/test/op_full_like_test.cpp

```diff
@@ -87,7 +87,7 @@ void OpFullLikeTest::test_full_like_out<ScalarType::Bool>() {
 
 TEST_F(OpFullLikeTest, AllRealOutputPasses) {
 #define TEST_ENTRY(ctype, dtype) test_full_like_out<ScalarType::dtype>();
-  ET_FORALL_REAL_TYPES_AND(Bool, TEST_ENTRY);
+  ET_FORALL_REALHBF16_TYPES(TEST_ENTRY);
```

Comment:
Missing the boolean. Use `ET_FORALL_REALHBBF16_TYPES`. Granted that the test name was misleading. Could you please change that as well. `AllRealOutputPasses` -> `AllDtypeOutputPasses`